cloudify_agent/tests/resources/blueprints/agent-from-package/scripts/create-package.py

```diff
@@ -25,7 +25,10 @@
 file_server_port = ctx.node.properties['file_server_port']
 
 package_name = create_agent_package(resource_base, config, ctx.logger)
-package_url = 'http://localhost:{0}/{1}'.format(file_server_port, package_name)
+package_url = 'http://localhost:{0}/{1}'.format(
```

Comment:
Checked it - this is a local file server, spun especially (and only) for the test. It is passed via the blueprint inputs to this script. So this is not a big.